js/FileManager.js

```diff
@@ -55,6 +57,9 @@ const FileManager = (() => {
 
         if (typeof ga !== 'undefined')
             ga('send', 'event', 'Pixel Editor Save', selectedPalette, currFile.canvasSize[0]+'/'+currFile.canvasSize[1]); /*global ga*/
+
+        
+        LayerList.closeOptionsMenu(); // is this the right place for this?
```

Comment:
I think it's unnecessary, it just closes the context menu for the layers, but at this point it should already be closed by all the other mouse events